contents/security/passwordStorage.md

```diff
@@ -0,0 +1,106 @@
+<frontmatter>
+  title: Introduction to Password Storage
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Password Storage
+
+Author: [Jeremy Choo](https://github.com/ChooJeremy)
+
+## Overview
+
+While developing your own application, it is very common to implement login systems to be able to identify users. Whenever there is a need to store user-specific information, developers typically implement user accounts to identify these users separately. You might have seen ways this are done in other websites - using registration forms and requring you to login with a username and password to identify you for the session.
```

Comment:
>seen ways this **are** done **in** other websites  

Grammatical error. Use `is` and `on` instead?

And I'd suggest that you rephrase it- are you trying to say that there are multiple different ways, or that it is done this way on other websites